### DISCUSSION TITLE
What mechanism(s) drives plate tectonics?

### DISCUSSION THREAD

**Discussion by LifeIsStrange** [a-cosmology-group/acg#265] [Astrophysics and Physics]
Plate tectonics which is part of geoscience which might seems off topic with astrophysics and cosmology but it is actually very much on topic and key to fundamental open questions. 
What is the age of earth? 
How did it form? 
What drives plate tectonics? 
It is essential because it alter the drake equation, fix a minimum age to the universe and the solar system, can answer the faint young sun problem and measurements of geoneutrinoes can inform us on theories of gravity, alter knowledge about particles, about conservation of energy and weather the earth has a nuclear reactor and even give insights on the sylurian hypothesis.
https://en.m.wikipedia.org/wiki/Earthquake_light

The standard model of plate tectonics is apparently, a very little known fact, falsified as the energy required for such distant mantle convection would require more energy than energy conservation allows, probably? even under the assumption of a georeactor. Though one might power plate tectonics via dark matter or neutrino capture but this is speculative new physics.
Cf https://arxiv.org/abs/1005.0738

The main alternative to the theory of plate tectonics is the expanding earth hypothesis, that seems to roughly work but powering an expansion of earth volume require an even bigger source of unknown energy.
Empirically, results are a bit discordant but measures of geoneutrinoes seems to shows that currently there is no nuclear or fusion reactor in the center of earth, or if there is, it would be weak, or the neutrinoes would be efficiently captured within earth which seems unlikely.
Also laser ranging seems to measure that earth IS currently expanding.
And indeed earth expansion is one of the only solutions to the faint young sun problem. 
IMO the most clever and underatted solution to the missing energy problem is to suppose that earth was formed and ejected (how?) from a forming giant planet (jupiter?) 
The genius of this idea is that this is already an existing alternative theory of planetary formation and that gravitational compression seems to lead to the right order of magnitude of compression of earth. 
Once earth has left its giant planet orbit, it would naturally decompress/expand over time, explaining plate tectonics without missing energy. 
The only other theory that seemed to be viable would be plate tectonics powered by expansion of some volume through the natural process of crystalisation, especially IIRC serpentinization. 
Btw a related topic is that we don't have a theory that explain and predict earthquakes. The only correlate that is partially successful is via maximal Lunar orbits through tidal effects (and maybe gravitational alterations, Cf the allais effect) 

https://phys.org/news/2024-08-dinosaur-footprints-sides-atlantic-ocean.html
It has recently been discovered an irrefutable proof via dinosaur footprints that south america separated from Africa 120 million years ago.
This timing probably rule out most expanding earth chronologies.

There is also the topic of polarization:
> Currently accepted engines for plate tectonics do not seem to supply sufficient energy for plate's motion and do not explain the globally observed asymmetries that from the Earth surface reach mantle depths.

> All this supports a general tuning of the Earth's geodynamics and mantle convection by astronomical forces.
https://www.sciencedirect.com/science/article/abs/pii/S0065268714000247

Btw there is also the slab pull mechanism but it seems insufficient or to require impossible convection? 

What do you think of it @expEarth ?

**Comment by ExpEarth** [comment]
`The standard model of plate tectonics is apparently, a very little known fact, falsified as the energy required for such distant mantle convection would require more energy than energy conservation allows, probably? even under the assumption of a georeactor. `

`The main alternative to the theory of plate tectonics is the expanding earth hypothesis, that seems to roughly work but powering an expansion of earth volume require an even bigger source of unknown energy.`

I've commented on these matters before, but am always happy to go over it again.  You're quite correct that heat sources presently thought to exist are not enough to drive either normal plate tectonics or Earth expansion.  However, I have previously described how both processes can arise due to the Hubble luminosity, the process that I also invoke in cosmology and gravity.  It is described by the simple relation _L = -UH_ , where _U_ is the internal gravitational potential energy of the mass or system and _H_ is the Hubble constant.

Here are some papers where I have described how slow Earth expansion works: 
[https://www.sciencedirect.com/science/article/pii/S1674984718301678](url)
[https://hgss.copernicus.org/articles/7/125/2016/](url)

`IMO the most clever and underatted solution to the missing energy problem is to suppose that earth was formed and ejected (how?) from a forming giant planet (jupiter?)`

This mechanism for Earth expansion energy is not likely.  The chemical composition of the Earth does not match that of the large planets like Jupiter.  We should look elsewhere to explain how the Earth was formed.  However, I can safely tell you that the ejection mechanism _is_ the correct one for how the Moon was formed (out of the Earth).

`It has recently been discovered an irrefutable proof via dinosaur footprints that south america separated from Africa 120 million years ago.
This timing probably rule out most expanding earth chronologies.`

It would only rule our fast expansion models and those can be ruled out anyway.  Only slow expansion works.

**Reply by HanDeBruijn** [reply]
@ExpEarth . When clicking on the above links I get

![fout](https://github.com/user-attachments/assets/0171cf33-ffed-48d2-befe-bdbce7bbb117)

It works when I replace them by
- [Deep mantle plumes and an increasing Earth radius](https://www.sciencedirect.com/science/article/pii/S1674984718301678)
- [Indications from space geodesy, gravimetry and seismology for slow Earth expansion at present – comment on “The Earth expansion theory and its transition from scientific hypothesis to pseudoscientific belief” by Sudiro (2014)](https://hgss.copernicus.org/articles/7/125/2016/)

Formatted as:
```
- [Deep mantle plumes and an increasing Earth radius](https://www.sciencedirect.com/science/article/pii/S1674984718301678)
- [Indications from space geodesy, gravimetry and seismology for slow Earth expansion at present – comment on “The Earth expansion theory and its transition from scientific hypothesis to pseudoscientific belief” by Sudiro (2014)](https://hgss.copernicus.org/articles/7/125/2016/)
```

**Comment by LifeIsStrange** [comment]
Discovery that earth mantle magma is most likely at least initially chemically homogeneous
https://phys.org/news/2024-09-envisions-earth-mantle-uniform-reservoir.html

**Comment by HanDeBruijn** [comment]
Too many updates for correcting too many errors. For that reason, a previous posting has been deleted and I decided to make a fresh start. 
___
Last night (CopyLefted 2024-09-19) I got another [wild](http://www.alternatievewiskunde.nl/cosmology/wild.htm) idea that might explain the Expanding Earth. The model is exploiting (my version of) the [Variable Mass Theory](https://www.youtube.com/watch?v=T7jVviVBzKs), as usual.

@LifeIsStrange .

>What is the age of earth?

In the Orbital timeframe $T$ it is : $-T=A_E = 4.543\times 10^9$ yr.

>How did it form?

By creation. How else would you define an **A**ge.

>What drives plate tectonics?

@ExpEarth .

>[ .. ] heat sources presently thought to exist are not enough to drive either normal plate tectonics or Earth expansion. [ .. ] **Only slow expansion works.**

It's related to the miracle of the [Shrinking Kilogram](http://www.alternatievewiskunde.nl/cosmology/narlikar.htm#feit). I'm asking my audience to ignore all of the "solutions" people have come up with. The only important thing here is _observation_, which can be summarized in two lines.

>A kilogram just isn't what it used to be. The 118-year-old cylinder that is the international prototype for the metric mass, kept tightly under lock and key outside Paris, is mysteriously losing weight - if ever so slightly.

>Physicist Richard Davis of the International Bureau of Weights and Measures in Sevres, southwest of Paris, says the reference kilo appears to have lost 50 micrograms compared with the average of dozens of copies.

Has the standard kilogram really been shrinking? No! Thanks to its **splendid isolation** it has remained constant all the time. But all matter around it has been suffering from increasing mass $m_0\to m$. The latter matter has become heavier, with an amount as calculated below, with Narlikar's law, for $m_0=1$ kg.

$$
\frac{m}{m_0} = \left(1+\frac{T}{A_E}\right)^2 \approx 1 + 2\frac{T}{A_E} \quad \Longrightarrow \quad \frac{m-m_0}{m_0} \approx 2\frac{T}{A_E} = \Delta m
$$

Ready to fill in the numbers.

$$
\Delta m = \frac{2 \times 118}{4.543\times 10^9} = 52 \times 10^{-9} \approx 50 \mbox{ micrograms}
$$

It makes no difference for such small time intervals if the Atomic timeframe $t$ is employed instead.

$$
\Delta m = \left(e^{t/A_E}\right)^2-1 \approx 2\frac{t}{A_E} \quad \mbox{with} \quad t \approx T
$$

An important contribution of @LifeIsStrange reveals
>that earth mantle magma is most likely at least initially chemically homogeneous

In space. Let's assume _that earth mantle magma_ is _homogeneous_ **in time** as well.
Then we have for the magma approximately the same situation as with the shrinking kilogram: splendid isolation, resulting in almost unchanged (elementary particle) mass.
The situation is quite different for the crust, however. Due to its enduring interaction with the rest of the cosmos, the change of its (elementary particle) mass is according to

$$
\frac{m}{m_0} = e^{H.t}
$$

where the Hubble parameter $H$ is calculated as
```
Mpc := 3.08567758*10^22;
H := 71*1000/Mpc;
```
Relevant for the size of atoms and molecules is the [Bohr radius](https://en.wikipedia.org/wiki/Bohr_radius). As can be easily derived, the Bohr radius $a$ varies inversely proportional to Variable Mass. See [Length Contraction](http://www.alternatievewiskunde.nl/cosmology/lengte.htm) at my website for more general results and details.
For $t=-A_E$ - at the time Earth was created - atoms and molecules were bigger than here and now:

$$
\frac{a_0}{a} = \frac{m}{m_0} = e^{H.t} = e^{-H.A_E} = 0.72
$$

Calculation details:
```
yr := 31556926;
A_E := 4.543*10^9*yr;
exp(-H*A_E);
```
Let the final shrunk area of the earth's crust here and now be denoted as $O_0$. Let the early area crust consisting of 100 % land be denoted as $O$. The quotient is

$$
\frac{O_0}{O} = \left(\frac{a_0}{a}\right)^2 = 0.5
$$

giving 50 % for "dry land", theoretically. Then it may be assumed that "water" has filled up the other 50 %. With google we find:
>water makes up about 71% of the Earth's surface, while the other 29% consists of continents and islands.

The above certainly is a very crude model. However, **NO spurious energy source is required**. Only the different behaviours of mantle and crust with respect to Variable Mass are taken into account. In this model, **the Expanding Earth is an illusion**. In reality, the mantle of the earth does not change much. Instead **the crust** - just 1 percent of Earth’s mass, an EggShell so to speak - **is shrinking**.

@LifeIsStrange :
>laser ranging seems to measure that earth IS currently expanding.

Of course. The laser **light**, according to VMT, is redshifted to the past, hence **blueshifted** to the future. Therefore distances **as measured** become **larger**, thus explaining the illusion that _earth IS currently expanding_. How much it **seems** to expand over $t=$ one year can be calculated.

$$
\frac{\lambda}{\lambda_0} = \frac{R}{R_0} = e^{H.t}
$$

where $\lambda=$ wavelength of laser light, $R=$ radius of the earth. It follows that

$$
R-R_0 = \Delta R \approx H.R_0.t  \approx 0.5 \mbox{ } mm / yr
$$

Calculation details:
```
R_0 := 6371*10^3;
R_0*H*yr;
```
[Accuracy of the International Terrestrial Reference Frame origin and Earth expansion](https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2011GL047450) is the only reference I could find on the internet with observations in it. From the Abstract: _the mean radius of the Earth is not changing to within_ $1\sigma$ _measurement uncertainty of_ $0.2\mbox{ }mm\mbox{ }yr^{−1}$.

![Growing_earth](https://github.com/user-attachments/assets/9a955846-d232-482f-b20e-2f4634cc95d6) to be replaced by ![gifmaker_me](https://github.com/user-attachments/assets/edc93ba9-27d7-4910-aa9e-d305f842dd56)

The model may be refined eventually by assuming a smooth transition between the "isolated" mantle and the non-isolated crust, instead of a sudden jump between the two. Or, as they usually say: _further research is needed_.

**Reply by ExpEarth** [reply]
I'm going to grant you points for creativity, Han.  A shrinking crust could mimic in appearance an expanding mantle.   But Occam's Razor might want to take those points away again, sorry to say.
